docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -88,6 +89,41 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
      (product, warehouse, location), (warehouse), (product), (warehouse, product), ())`.
     The N elements of a `CUBE` specification results in 2^N `GROUPING SETS`.
 
+* **Partial Grouping Analytics**
+
+    Partial grouping analytics means there are both `group_expression` and `CUBE|ROLLUP|GROUPING SETS`
+    in GROUP BY clause. For example:
+    `GROUP BY warehouse, CUBE(product, location)` is equivalent to
+    `GROUP BY GROUPING SETS((warehouse, product, location), (warehouse, product), (warehouse, location), (warehouse))`.
+    `GROUP BY warehouse, ROLLUP(product, location)` is equivalent to
+    `GROUP BY GROUPING SETS((warehouse, product, location), (warehouse, product), (warehouse))`.
+    `GROUP BY warehouse, GROUPING SETS((product, location), (producet), ())` is equivalent to
+    `GROUP BY GROUPING SETS((warehouse, product, location), (warehouse, location), (warehouse))`.
+
+* **Concatenated Groupings**
+  
+    Concatenated grouping analytics offer a concise way to generate useful combinations of groupings. Groupings specified
+    with concatenated groupings yield the cross-product of groupings from each grouping set. The cross-product 
+    operation enables even a small number of concatenated groupings to generate a large number of final groups. 
+    The concatenated groupings are specified simply by listing multiple `GROUPING SETS`, `CUBES`, and `ROLLUP`, 
```

Comment:
We can just say that CUBE/ROLLUP are just syntax sugar for GROUPING SETS, then the document can just focus on GROUPING SETS.

Reply:
> We can just say that CUBE/ROLLUP are just syntax sugar for GROUPING SETS, then the document can just focus on GROUPING SETS.

Done